docs/wiki/01 - STUDY PARAMETERS SETUP/01 - Study Variables/04 - Subprojects.md

```diff
@@ -0,0 +1,46 @@
+# Subprojects
+
+## Overview
+The **Subproject** plays an important role in determining the instrument battery that will be assigned to a candidate at a timepoint. **Subprojects** are defined in the `subproject` table of the database.
```

Comment:
See suggested edit below, covering 2 points: 
1. `defined` is too vague.  I moved the basic "where are they, and how do I add them" to the top. (you may need to clean the text later re Config module)
2. "plays a role" is too vague.  Let's actually articulate the relationship -- something like:
```suggestion
**Subprojects** are defined in the front-end by the Configuration module, and are stored in the `subproject` table of the database.
The **Subproject** plays an important role in determining the instrument battery that will be assigned to a candidate at a timepoint.  The subproject can be used as a variable when determining which instruments are populated -- this allows for subproject-specific batteries to be defined in the `test_battery` table. 
```